client/src/draft/DraftState.ts

```diff
@@ -50,6 +50,7 @@ export interface MtgCard {
   collector_number: string;
   cmc: number;
   color: string;
+  mtgo: string;
```

Comment:
Can you add a comment describing what this is (I assume MTGO ID?) Or rename it to be mgtoId or something else more descriptive.